catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -1478,4 +1592,41 @@ void setUri(UriInfo uri) {
   Bundle getBundle() {
     return FrameworkUtil.getBundle(this.getClass());
   }
+
+  public void init() {
+    int numThreads = 2 * Runtime.getRuntime().availableProcessors();
```

Comment:
Should we add a system property for this in case it needs to be adjusted in an operational environment? Some of our other threadpools we do similiar with.